app/models/cms/builtin/legal_term.rb

```diff
@@ -29,13 +31,13 @@ def find_or_build_by_system_name(system_name, attributes)
       end
   end
 
-  def save(*)
-    self.published = self.draft
-    super
-  end
-
   def content_type
     "text/html"
   end
 
+  private
+
+  def publish_draft
+    self.published = draft unless published
```

Comment:
in the original we always publish on save, not checking `published` :thinking:

Reply:
Hmmm... I don't remember the details of why I did that exactly. But I think it's better to do it in hooks than override the default methods.

There was a reason for this `unless published` though. Let me think.

Reply:
Thanks for catching! This is actually a bug I introduced. I couldn't remember why I did it (I will probably figure out in some failing tests), but this `unless published` should be indeed removed.